backends/jwt_test.go

```diff
@@ -1654,3 +1654,77 @@ func TestJWTFormTextResponseServer(t *testing.T) {
 	})
 
 }
+
+func TestJWTHttpDelayedResponse(t *testing.T) {
+	topic := "test/topic"
+	var acc = int64(1)
+	clientID := "test_client"
+	token, _ := jwtToken.SignedString([]byte(jwtSecret))
+	// wrongToken, _ := wrongJwtToken.SignedString([]byte(jwtSecret))
+
+	version := "2.0.0"
+
+	rightToken := token
+
+
+	mockServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		time.Sleep(5 * time.Second)
+
+		err := r.ParseForm()
+		if err != nil {
+			w.WriteHeader(http.StatusBadRequest)
+			return
+		}
+		var params = r.Form
+
+		gToken := r.Header.Get("Authorization")
+		gToken = strings.TrimPrefix(gToken, "Bearer ")
+
+		if rightToken != gToken {
+			w.WriteHeader(http.StatusNotFound)
+			return
+		}
+
+		switch r.URL.Path {
+		case "/user", "/superuser":
+			w.WriteHeader(http.StatusOK)
+		case "/acl":
+			paramsAcc, _ := strconv.ParseInt(params["acc"][0], 10, 64)
+			if params["topic"][0] == topic && params["clientid"][0] == clientID && paramsAcc <= acc {
+				w.WriteHeader(http.StatusOK)
+				break
+			}
+			w.WriteHeader(http.StatusNotFound)
+		}
+
+	}))
+
+	defer mockServer.Close()
+
+
```

Comment:
```suggestion

```

Reply:
I could spend sometime in doing this PR.. Let me know.. I am unsure if `gofmt` is used well from my side. The tests are ok right now. Let me know. Thank you for your suggestions :)